tests/loggers/test_object_store_logger.py

```diff
@@ -9,7 +9,8 @@
 
 from composer.core.event import Event
 from composer.core.state import State
-from composer.loggers import Logger, LogLevel, ObjectStoreLoggerHparams
+from composer.loggers import Logger, LogLevel
+from composer.loggers.logger_hparams import ObjectStoreLoggerHparams
```

Comment:
Still depends on Hparams tho? I thought this PR was supposed to remove dependencies to hparams?

Reply:
I decided to leave the `ObjectStoreLoggerHparams`, as it does some special parsing to load secrets from env variables (to discourage storing of secrets in config).